README.md

```diff
@@ -23,7 +23,7 @@ aws-security-viz -- A tool to visualize aws security groups
 
 ## DEPENDENCIES
 
-* graphviz with triangulation `brew install graphviz --with-gts`
+* graphviz with triangulation `brew install graphviz`
```

Comment:
The GTS option is what would add the triangulation support. Given the option doesn't exist anymore, I guess it might be better to just say 

* graphviz `brew install graphviz`

Reply:
`graphviz` seems [to depend](https://formulae.brew.sh/formula/graphviz) on gts now.

I'll make the text change.